features/features.go

```diff
@@ -44,14 +44,6 @@ var (
 					enhancementPR("https://github.com/openshift/enhancements/pull/1706").
 					mustRegister()
 
-	FeatureGateServiceAccountTokenNodeBinding = newFeatureGate("ServiceAccountTokenNodeBinding").
-							reportProblemsToJiraComponent("apiserver-auth").
-							contactPerson("stlaz").
-							productScope(kubernetes).
-							enhancementPR("https://github.com/kubernetes/enhancements/issues/4193").
-							enableIn(configv1.DevPreviewNoUpgrade, configv1.TechPreviewNoUpgrade).
-							mustRegister()
```

Comment:
This gate has been removed in Kubernetes 1.33, so we need to drop it here.
CC @ibihim

Reply:
Drop it? you mean we remove it completely, instead of adding `configv1.Default`?
Hm, means we would need to drop the feature gate, wherever it is being used?